Fabric.Identity.API/Configuration/IdentityConfigurationProvider.cs

```diff
@@ -1,19 +1,15 @@
-﻿using System.Security.Cryptography;
-using System.Security.Cryptography.X509Certificates;
-using Fabric.Identity.API.Services;
+﻿using Fabric.Identity.API.Services;
```

Comment:
Moving using directives inside the namespace scope would avoid naming issues.  However, because of the style of the application, and there are no conflicts, its probably save to keep these as is.